data/human/human missions.txt

```diff
@@ -5428,3 +5428,71 @@ mission "Deal Change 2"
 				die
 			label skip
 			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`
+
+
+
+mission "Quicksilver Mixup [2]"
+	name "Misposted mail delivery"
+	description "Carry misposted mail from Silver to Quicksilver by <date> and then come back with the return shipment"
+	minor
+	deadline 10 1
+	source "Silver"
+	destination "Quicksilver"
+	cargo "Misposted mail" 38
+	to offer
+		has "Quicksilver Mixup [1]: done"
+		has "quicksilver mixup 2 available"
+	on offer
+		conversation
+			`As you walk through Silver's spaceport, somebody wearing office attire power walks up to you after ending a hurried conversation with another pilot.`
+			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to Quicksilver and back couldn't make it this month, some mechanical issues with his ship, I'm told. Would you be willing to carry this month's load there by <day>? We'll pay you what he'd be paid, 100.000 credits."`
```

Comment:
Is there any advantage or disadvantage to having it written one way or the other? Seems like it would allow you to edit the payment by changing one as opposed to two lines, without cons. I'd say this is a fine change, right?

Either way, i will wait for a third person to check before committing any of these, as you asked.

Reply:
There's no disadvantage as far as I know.

Reply:
Thank you, i'll commit it then.